uaclient/tests/test_util.py

```diff
@@ -596,7 +596,12 @@ def test_fallback_used_if_all_other_files_are_empty(
 class TestIsServiceUrl:
     @pytest.mark.parametrize(
         "url,is_valid",
-        (("http://asdf", True), ("http://asdf/", True), ("asdf", False)),
+        (
+            ("http://asdf", True),
+            ("http://asdf/", True),
```

Comment:
Maybe we can add a successful port example in here too?
```suggestion
            ("http://asdf/", True),
            ("http://asdf:1234", True),
```